app/models/submission_state_actions.rb

```diff
@@ -15,6 +15,8 @@ def run
     case submission.state
     when Submission::DRAFT, Submission::SUBMITTED
       [approve_action, publish_action, hold_action, reject_action, close_action]
+    when Submission::RESUBMITTED
+      [publish_action, hold_action, reject_action, close_action]
```

Comment:
Noticing that we don't allow a submission to transition to `rejected` once it has been `approved`. Since this new `resubmitted` state comes after `approved`, should the same rule apply? Perhaps the only allowed options should be publish/hold/close.

Reply:
Good question, is it possible that info provided in tier 2 "invalidates" the whole submission? For instance, user initially specified location as "Germany" (in tier 1), but then specified some country in tier 2 from which we cannot organize artwork delivery. I'm just speculating - don't know if such case is real.

Also user could change something even in tier 1 information list - I think in this cases we should allow rejecting such submissions

Reply:
In that case, maybe the submission is `closed` instead? I feel like the approval/rejection decision falls at the same point in the internal process and it might be awkward to have a submission go from submitted > approved > rejected based on how that process is currently defined.